libcxx/include/__iterator/wrap_iter.h

```diff
@@ -24,6 +24,7 @@
 #include <__type_traits/is_convertible.h>
 #include <__type_traits/is_same.h>
 #include <__type_traits/make_const_lvalue_ref.h>
+#include <__utility/pair.h>
```

Comment:
Why is this required? AFAICT `pair` isn't used in this file.

Reply:
I can't seem to reproduce right now since I'm getting an unrelated bug where clang segfaults when you provide multiple module maps, but I explicitly remember having the exact same thought, but it was required for something or other and would fail to compile without it.

Might have been deduction guides? I forget exactly what it was.